exercises/series/.meta/src/reference/java/Series.java

```diff
@@ -17,7 +17,7 @@
     }
 
     List<List<Integer>> slices(int num) {
-        if (num > this.digitsSize) {
+        if (num > this.digitsSize || num == 0) {
             throw new IllegalArgumentException("Slice size is too big.");
```

Comment:
I don't think this is the correct exception message for `num == 0`? 🙂

Reply:
done